src/aux-records/AIController.spec.ts

```diff
@@ -369,6 +369,111 @@ describe('AIController', () => {
             expect(chatInterface.chat).not.toBeCalled();
         });
 
+        it('should return success when allowedModels includes the model', async () => {
+            chatInterface.chat.mockReturnValueOnce(
+                Promise.resolve({
+                    choices: [
+                        {
+                            role: 'user',
+                            content: 'test',
+                            finishReason: 'stop',
+                        },
+                    ],
+                    totalTokens: 1,
+                })
+            );
+
+            const result = await controller.chat({
+                model: 'test-model1',
+                messages: [
+                    {
+                        role: 'user',
+                        content: 'test',
+                    },
+                ],
+                temperature: 0.5,
+                userId,
+                userSubscriptionTier,
+            });
```

Comment:
@TroyceGowdy This test should be inside the "subscriptions" group and it should setup a subscription configuration that specifies the allowed models for the subscription tier that the user has. See [lines 626-649](https://github.com/casual-simulation/casualos/blob/2cbb37a2ed040cf82f9960f8e06e52ae943bb455/src/aux-records/AIController.spec.ts#L626-L649) for an example of setting up the subscription configuration.

The way this test is currently written, it doesn't actually test the new subscription feature limit and only tests the server config.